python/tidy/servo_tidy/tidy.py

```diff
@@ -167,7 +156,8 @@ def licensed_apache(header):
 
 
 def check_license(file_name, lines):
-    if any(file_name.endswith(ext) for ext in (".toml", ".lock", ".json", ".html")):
+    if any(file_name.endswith(ext) for ext in (".toml", ".lock", ".json", ".html")) or \
```

Comment:
This would read better if you structured it like

```
if not config["skip-check-licenses"]:
    if any(file_name.endswith(ext) for ext in (".toml", ".lock", ".json", ".html")):
          raise StopIteration 
```

To simplify the double negative in the first condition, you could optionally change "skip-check-licenses" to "check-licenses" and default it to true.